pkg/runtime/application_config_provider.go

```diff
@@ -38,6 +38,11 @@ var flyteAdminConfig = config.MustRegisterSection(flyteAdmin, &interfaces.Applic
 	EventVersion:          2,
 	AsyncEventsBufferSize: 100,
 	MaxParallelism:        25,
+	Labels:                map[string]string{"defaultFlyteLabelKey": "defaultFlyteLabelValue"},
+	Annotations:           map[string]string{"defaultFlyteAnnotationKey": "defaultFlyteAnnotationValue"},
+	OutputLocationPrefix:  "defaultFlyteOutputLocationPrefix",
+	K8SServiceAccount:     "default",
+	AssumableIamRole:      "default",
```

Comment:
Are we ok with these defaults @katrogan

Reply:
Hm, besides setting the `K8SServiceAccount` are the rest of these actually useful for a flyte execution? maybe we can omit the rest